.github/workflows/ci.repos

```diff
@@ -0,0 +1,17 @@
+repositories:
```

Comment:
amlip missing?

Reply:
CI.repos does not need to download amlip as it is already taken from github